recipe-server/normandy/recipes/api/views.py

```diff
@@ -120,7 +120,13 @@ def history(self, request, pk=None):
     def enable(self, request, pk=None):
         recipe = self.get_object()
         recipe.enabled = True
-        recipe.save()
+
+        try:
+            recipe.save()
+        except Recipe.NotApproved:
+            return Response({'enabled': 'Cannot enable a recipe that has not been approved.'},
```

Comment:
Do we want to use the message from the exception object, instead of duplicating it here?